plugins/trivy/cmd/trivy/main.go

```diff
@@ -100,4 +118,47 @@ func main() {
 	if err != nil {
 		panic(err)
 	}
+	logrus.Info("Finished writing file ", outputFile)
+}
+
+func getNewestVersionsToScan(ctx context.Context, imageWithVulns []models.ImageReport) []models.Image {
+	versionsChan := make(chan newestVersions, len(imageWithVulns))
+	for _, i := range imageWithVulns {
```

Comment:
is it possible that there are duplicate images/repositories in here? we might be double-scanning if so